components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -144,6 +149,24 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
                         />
                     </button>
                 );
+
+                if (minimumSKURequiredForFeature === LicenseSkus.Enterprise) {
+                    ctaPrimaryButton = (
+                        <button
+                            className='btn btn-primary'
+                            data-testid='featureDiscovery_primaryCallToAction'
+                            onClick={() => {
+                                trackEvent(TELEMETRY_CATEGORIES.CLOUD_ADMIN, 'click_enterprise_contact_sales_feature_discovery');
+                                window.open(contactSalesLink, '_blank');
```

Comment:
Leaving the comment here as discussed offline, we need to support the contact sales link for the case of self-managed here

Reply:
Just pushed the fix for that